pkg/utils/logutils/logutils.go

```diff
@@ -0,0 +1,56 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logutils
```

Comment:
I think this should be added to **admiral** so it could be used by the other repos.

Reply:
Yes, I agree, just wanted to get initial approval for integrating in a single component. Once we decide to integrate in all components I will move it to `admiral`.